### PR TITLE
Added contributing.md and contributors.md

### PR DESCRIPTION
**Pull request by AReyH**
Added the CONTRIBUTORS.md containing all contributing members of the AutoPredict project, and added CONTRIBUTING.md which explains how to contribute to the AutoPredictor project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,73 +1,30 @@
...
+-   Pull Requests will be reviewed as fast as our team can look at them. Please be mindful that we are students, and our schedule can vary a bit.
+-   New code should follow the PEP 8 styleguide.
 
-You can never have enough documentation! Please feel free to contribute to any
-part of the documentation, such as the official docs, docstrings, or even
-on the web in blog posts, articles, and such.
+### Code of Conduct
 
-### Submit Feedback
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
```
Comment (zgarciaj) [inline CONTRIBUTING.md]:
```suggestion
Please note that this project is released with a [Contributor Code of Conduct](https://github.com/UBC-MDS/autopredictor2/blob/main/CONDUCT.md). By participating in this project you agree to abide by its terms.
```

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+### **Contributors:**
+
+* Anu Banga
```
Comment (zgarciaj) [inline CONTRIBUTORS.md]:
```suggestion
* Anu Banga - @AnuBanga 
```

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+* Zeily Garcia
```
Comment (zgarciaj) [inline CONTRIBUTORS.md]:
```suggestion
* Zeily Garcia - @zgarciaj 
```

File: CONTRIBUTORS.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+* Arturo Rey Haggar
```
Comment (zgarciaj) [inline CONTRIBUTORS.md]:
```suggestion
* Arturo Rey Haggar - @AReyH 
```

File: CONTRIBUTORS.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+* Sharon Voon
```
Comment (zgarciaj) [inline CONTRIBUTORS.md]:
```suggestion
* Sharon Voon - @s-voon 
```

**Review by zgarciaj — COMMENTED** [review]
Excellent work, Arturo! I have proposed a few minor adjustments, such as incorporating our GitHub usernames into the CONTRIBUTORS.md file. Additionally, I have recommended an updated hyperlink that directs to the CONDUCT.md file path. Nonetheless, should we decide to rename the repository (to autopredictor instead of autopredictor2), it will be necessary to revise this suggested link accordingly.

File: CONTRIBUTING.md (hunk 1 above)
Comment (AReyH) [reply CONTRIBUTING.md:26]: no.

**Review by zgarciaj — APPROVED** [review]
Well done!

**Pull request merged**